### PR TITLE
Add CAN GPS

### PR DESCRIPTION
**Pull request by davids5**
Waiting on upstream to round trip backport. - pointing to branch

Todos:
- [x] PR review comments
- [x] Add Flash parameters 
- [ ] Support USB 
- [ ] Add PPS driver

Given the above, the bootloader code here in should not be deployed. The bootloader size and Application code location may HAVE to change.

This branch should only be used for HW verification.

### CODE REVIEW

File: boards/holybro/can-gps-v1/init/rc.board_defaults [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+#
+# board specific defaults
+#------------------------------------------------------------------------------
+
+
+if [ $AUTOCNF = yes ]
+then
+	# Disable safety switch by default
+	param set CBRK_IO_SAFETY 22027
```
Comment (dagar) [inline boards/holybro/can-gps-v1/init/rc.board_defaults:10]: We shouldn't set this param.

File: boards/holybro/can-gps-v1/src/Makefile [hunk 2]
```diff
@@ -0,0 +1,100 @@
+############################################################################
```
Comment (dagar) [inline boards/holybro/can-gps-v1/src/Makefile:1]: delete?

**Comment by dagar** [comment]
We might as well continue incremental in master.

**Review by dagar — APPROVED** [review]

**Comment by ShiauweiZhao** [comment]
Will gps and gps utc time be used to calibrate px4 systemtime?

**Pull request merged**